src/DeVinci_frontend/components/SidebarBottomNav.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+export let currentModel;
+
+currentModel = "Hermes 2 Pro Llama4";
```

Comment:
you can get the current model like so:
store exports selectedAiModelId
pass selectedAiModelId to a new function exported in ai_model_helpers.js (the functionality from getDefaultAiModelId can be copied and slightly adjusted to do the trick)

Reply:
added to development